apps/core/app/[locale]/(default)/cart/_components/shipping-info.tsx

```diff
@@ -142,6 +142,7 @@ export const ShippingInfo = ({
           <FieldLabel>{t('country')}</FieldLabel>
           <FieldControl asChild>
             <Select
+              aria-label={t('countryAriaLabel')}
```

Comment:
Looking above, it doesn't look like there is an `htmlFor` + `id` relationship. The best practice for accessibility is to use `aria-*` attributes as a last resort, specifically when there are no other native features to initially use.

That being said, I think we need to augment our `Select` component to accept an `id` prop and forward that attribute to the trigger. That way we can add an identifier to the field in order to attribute the `<label>` above. 

(Same with the select below)